includes/connection/class-products.php

```diff
@@ -379,13 +379,19 @@ public static function get_connection_args( $extra_args = [] ): array {
 				'type'        => 'Int',
 				'description' => __( 'Limit result set to products assigned a specific shipping class ID.', 'wp-graphql-woocommerce' ),
 			],
+			'attributes'         => [
+				'type'        => 'ProductAttributeQueryInput',
+				'description' => __( 'Limit result set to products with selected global attribute queries.', 'wp-graphql-woocommerce' ),
+			],
 			'attribute'          => [
-				'type'        => 'String',
-				'description' => __( 'Limit result set to products with a specific attribute. Use the taxonomy name/attribute slug.', 'wp-graphql-woocommerce' ),
+				'type'              => 'ProductAttributeEnum',
```

Comment:
Changing the type makes this breaking for schemas already using it. If it's deprecated anyway is there a need to change the type as well?

Reply:
No, there isn't thanks for letting me know. I'll change it back before merging